doc/cascadia/profiles.schema.json

```diff
@@ -817,6 +819,63 @@
         }
       ]
     },
+    "GlobalSummonAction": {
+      "description": "Arguments corresponding to a focusPane Action",
+      "allOf": [
+        { "$ref": "#/definitions/ShortcutAction" },
+        {
+          "properties": {
+            "action": { "type": "string", "pattern": "globalSummon" },
+            "desktop":{
```

Comment:
Elsewhere in the file, it seems the style is to add a space between the colon and the value.